cmd/plume/prerelease.go

```diff
@@ -699,30 +724,23 @@ func awsPreRelease(ctx context.Context, system string, client *http.Client, src
 		return nil
 	}
 
-	imageFileNameTmpl := spec.AWS.Image
-	imageName := fmt.Sprintf("%v-%v-%v", spec.AWS.BaseName, specChannel, specVersion)
-	imageName = regexp.MustCompile(`[^A-Za-z0-9()\\./_-]`).ReplaceAllLiteralString(imageName, "_")
-	imageDescription := fmt.Sprintf("%v %v %v", spec.AWS.BaseDescription, specChannel, specVersion)
-
-	if system == "fedora" {
-		t := template.Must(template.New("filename").Parse(imageFileNameTmpl))
-		buffer := &bytes.Buffer{}
-		if err := t.Execute(buffer, imageMetadata); err != nil {
-			panic(err)
-		}
-		imageFileName := buffer.String()
-		imageName = strings.TrimSuffix(imageFileName, ".raw.xz")
-		imageDescription = fmt.Sprintf("%v %v %v", spec.AWS.BaseDescription, specChannel, specVersion)
+	imageFileName, imageName, imageDescription := getSpecAWSImageName(spec, imageMetadata)
+	imagePath, err := getImageFile(system, client, src, imageFileName)
+	if err != nil {
+		return err
 	}
 
-	imagePath, err := getImageFile(system, client, src, imageFileNameTmpl)
+	vmdkImagePath := strings.Replace(imagePath, "raw", "vmdk", 1)
```

Comment:
Isn't this block of code going to run on CL images as well which are already `vmdk` images?

Reply:
I updated this part of code later in https://github.com/coreos/mantle/pull/934/commits/c7197d25e571317664032ec0dd32dec260f3f2e2